app/views/base/index.html.erb

```diff
@@ -28,9 +26,9 @@
           </section>
         <% end %>
         <section>
-          <a class="call-to-action" href="<%= root_path %>">
-            <%= t('landing.preview.call-to-action') %>
-          </a>
+          <div class="link">
```

Comment:
Is this class necessary? I don't see it being used for styling but I could be missing something